### ISSUE TITLE
Upgrade `winit`/`raw-window-handle` to latest version

### ISSUE THREAD

**Issue by hannobraun** [hannobraun/Fornjot#845]
We currently depend on `raw-window-handle` 0.4.3. As of this writing, the latest version is 0.5.0.

Upgrading is currently blocked on [`winit`](https://crates.io/crates/winit) 0.27.0 being published (https://github.com/rust-windowing/winit/pull/2376), and `wgpu` (which doesn't seem to have started the upgrade).

**Comment by hannobraun** [comment]
No longer blocked on winit. Still blocked on wgpu, which has an open pull request now (https://github.com/gfx-rs/wgpu/pull/2918). Updated the issue title accordingly.

**Comment by hannobraun** [comment]
Winit 0.27.2 supports `raw-window-handle` 0.4, which means it should work with wgpu. No longer blocked!

**Comment by hannobraun** [comment]
Closing in favor of #976.
